source/include/vx_reference.h

```diff
@@ -196,6 +211,18 @@ vx_uint32 ownIncrementReference(vx_reference ref, vx_enum reftype);
  */
 vx_uint32 ownDecrementReference(vx_reference ref, vx_enum reftype);
 
+/*! \brief Returns the total reference count of the object.
+ * \param [in] ref The reference to print.
+ * \ingroup group_vx_reference
+ */
+vx_uint32 ownTotalReferenceCount(vx_reference ref);
```

Comment:
I don't see any other reference to this function in the code ... I think this can be removed.

Reply:
removed